app/jobs/hackathons/send_organizer_summaries_job.rb

```diff
@@ -0,0 +1,19 @@
+class Hackathons::SendOrganizerSummariesJob < ApplicationJob
+  def perform(sent_digests)
+    # This reloads the (possible) sent_digests array as an
+    # ActiveRecord::Relation so that we can use includes to prevent an N+1.
+    @sent_digests = Hackathon::Digest.where(id: sent_digests.map(&:id)) ||
+      Hackathon::Digest.where(created_at: 6.days.ago...Time.now)
```

Comment:
This conditional isn't needed.


```suggestion
    @sent_digests = Hackathon::Digest.where(id: sent_digests.map(&:id))
```